### PR TITLE
댓글 좋아요 gql

### PR DESCRIPTION
**Pull request by robin-maki**

### CODE REVIEW

**Comment by robin-maki** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/penxle/penxle/1436?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* **#1436** <a href="https://app.graphite.dev/github/pr/penxle/penxle/1436?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1435** <a href="https://app.graphite.dev/github/pr/penxle/penxle/1435?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1434** <a href="https://app.graphite.dev/github/pr/penxle/penxle/1434?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/penxle/penxle/1436?utm_source=stack-comment).

File: apps/penxle.com/src/lib/server/graphql/schemas/comment.ts [hunk 1]
```diff
@@ -51,6 +51,25 @@ export const commentSchema = defineSchema((builder) => {
         },
       }),
 
+      likeCount: t.relationCount('likes'),
+      likedByWriter: t.boolean({
```
Comment (devunt) [inline apps/penxle.com/src/lib/server/graphql/schemas/comment.ts:55]: P3: 우리 writer 라는 단어 쓰나....요?

Comment (robin-maki) [reply apps/penxle.com/src/lib/server/graphql/schemas/comment.ts:55]: 지금 확인해보니까 확실히 안 쓰긴 하네요... 근데 뭘로 이름을 지어야 할 지 잘 모르겠어요

Comment (devunt) [reply apps/penxle.com/src/lib/server/graphql/schemas/comment.ts:55]: 흠 저도 고민되긴 하네요... `likedByPostedUser`? `likedByPostAuthor`?

Comment (robin-maki) [reply apps/penxle.com/src/lib/server/graphql/schemas/comment.ts:55]: Author 역시 우리가 평소에 쓰진 않았어서 `likedByPostedUser`가 가장 무난할 거 같아요

**Review by devunt — APPROVED** [review]

**Pull request merged**